Frameworks/UIKit.Xaml/Button.xaml

```diff
@@ -5,21 +5,15 @@
     xmlns:local="using:UIKit.Xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d"
-    Width="50"
-    Height="50">
+    mc:Ignorable="d">
     <Button.Template>
         <ControlTemplate TargetType="Button" x:Name="UIKitButtonControlTemplate">
-            <Grid x:Name="rootGrid" Background="{TemplateBinding Background}">
-                <!-- TODO: Since NineGrid works on image pixels, to support an image with 2x scaling we need to multiply the Size and NineGrid, 
-                     and inverse render transform it back to the right size. Grid clips its children, so we need to host it in a Canvas first. -->
-                <Image x:Name="backgroundImage" 
-                       Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" IsHitTestVisible="False" Stretch="Fill"/>
+            <Canvas x:Name="rootCanvas" Background="{TemplateBinding Background}">
                 <Canvas x:Name="contentCanvas" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}">
```

Comment:
any thoughts on this? it definitely seems like we can consolidate this to a single canvas.

---
In reply to: [87646076](https://github.com/Microsoft/WinObjC/pull/1361#discussion_r87646076) [](ancestors = 87646076)